### ISSUE TITLE
All boolean variables are default to `true` in rc.75

### ISSUE THREAD

**Issue by t7tran** [directus/directus#6196]
1) The issue and what you expected to happen
    Directus app should use the default environment variables declared in `env.ts`'s defaults. However, it is observed that all boolean variables are all set to `true` regardless of the actual environmental values. Affected variables are:

```
RATE_LIMITER_ENABLED
REFRESH_TOKEN_COOKIE_SECURE
CORS_ENABLED
CORS_ORIGIN
CORS_CREDENTIALS
CACHE_ENABLED
TELEMETRY
```

2) Exact steps to reproduce this issue
    Just start up Directus without any custom configuration and keep refreshing the page, it would eventually render `REQUESTS_EXCEEDED` error even though `RATE_LIMITER_ENABLED` is disabled by default.

3) Your environment:
    Directus rc.75
    MySQL 5.7
    Docker
    Latest Chrome

4) Any other relevant information we might need to reproduce this issue
    N/A

**Comment by t7tran** [comment]
Possibly caused by #6101

**Comment by paescuj** [comment]
Duplicate of #6189 and fixed in #6190

**Comment by nickrum** [comment]
Duplicate of #6189
